<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DepthMapCalibrator</class>
 <widget class="QMainWindow" name="DepthMapCalibrator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>555</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>30</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>numDisparties</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>70</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Block Size</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>70</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>7</number>
    </property>
    <property name="maximum">
     <number>255</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="pageStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Disp12MaxDiff</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_3">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>110</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Specle Range</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_4">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>14</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Uniqueness Ratio</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_5">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>210</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Texture Treshold</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_6">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>13</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Min Disparity</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_7">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>290</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="value">
     <number>-3</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>330</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Prefilter Cap</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_8">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>330</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>62</number>
    </property>
    <property name="value">
     <number>29</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>370</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Prefilter Size</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_9">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>370</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>7</number>
    </property>
    <property name="value">
     <number>7</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Specle Window Size</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider_10">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>160</width>
      <height>21</height>
     </rect>
    </property>
    <property name="value">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>70</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>150</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>250</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>290</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>330</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>370</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>510</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>horizontalSlider</sender>
   <signal>sliderReleased()</signal>
   <receiver>DepthMapCalibrator</receiver>
   <slot>updateValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>62</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_9</sender>
   <signal>sliderReleased()</signal>
   <receiver>DepthMapCalibrator</receiver>
   <slot>updateValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>382</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_8</sender>
   <signal>sliderReleased()</signal>
   <receiver>DepthMapCalibrator</receiver>
   <slot>updateValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>342</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_7</sender>
   <signal>sliderReleased()</signal>
   <receiver>DepthMapCalibrator</receiver>
   <slot>updateValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_6</sender>
   <signal>sliderReleased()</signal>
   <receiver>DepthMapCalibrator</receiver>
   <slot>updateValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_5</sender>
   <signal>sliderReleased()</signal>
   <receiver>DepthMapCalibrator</receiver>
   <slot>updateValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_4</sender>
   <signal>sliderReleased()</signal>
   <receiver>DepthMapCalibrator</receiver>
   <slot>updateValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_3</sender>
   <signal>sliderReleased()</signal>
   <receiver>DepthMapCalibrator</receiver>
   <slot>updateValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_2</sender>
   <signal>sliderReleased()</signal>
   <receiver>DepthMapCalibrator</receiver>
   <slot>updateValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>216</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_10</sender>
   <signal>sliderReleased()</signal>
   <receiver>DepthMapCalibrator</receiver>
   <slot>updateValues()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>277</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>updateValues()</slot>
 </slots>
</ui>
